tools/clang/include/clang/SPIRV/EmitVisitor.h

```diff
@@ -171,8 +173,10 @@ class EmitVisitor : public Visitor {
   // using the type information.
 
 private:
+  // The next available result-id to be assigned.
+  uint32_t id;
```

Comment:
Actually this holds the previously used id?

Reply:
Oh right. Updated the comment.